dm/tests/sync_collation/run.sh

```diff
@@ -0,0 +1,191 @@
+#!/bin/bash
+
+set -eu
+
+cur=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+source $cur/../_utils/test_prepare
+WORK_DIR=$TEST_DIR/$TEST_NAME
+db="sync_collation"
+db_increment="sync_collation_increment"
+db2="sync_collation2"
+db_increment2="sync_collation_increment2"
+tb="t"
+tb2="t2"
+tb_check="t_check"
+
+function run() {
```

Comment:
I think we can only have one "all" mode task, so many lines below can be removed

Reply:
we can use "all" mode instead "full" and "incremental" mode by whether or not run 'xxx.increment.sql'. i will fix.